### DISCUSSION TITLE
language

### DISCUSSION THREAD

**Discussion by ankursatija20** [twbs/bootstrap#37514] [Q&A]
how to change arabic to engiish in code

**Comment by TianMeds** [comment]
To open the Command Palette, use Ctrl+Shift+P, then start entering "display" to filter and show the Configure Display Language command. this is applicable in vscode you can visit the link here: https://code.visualstudio.com/docs/getstarted/locales#:~:text=Press%20Ctrl%2BShift%2BP%20to,with%20the%20current%20locale%20highlighted.
